openmls/src/messages/proposals.rs

```diff
@@ -183,7 +183,7 @@ pub struct AddProposal {
 
 impl AddProposal {
     /// Get a reference to the key package in the proposal.
-    pub(crate) fn key_package(&self) -> &KeyPackage {
+    pub fn key_package(&self) -> &KeyPackage {
```

Comment:
Do I understand correctly, that those functions need to be public now for inspection by the consumer?

Reply:
Yes, consumers can now inspect this through the `StagedCommit` they might receive when processing messages.